cmd/govim/main.go

```diff
@@ -246,7 +246,12 @@ func (g *govimplugin) Init(gg govim.Govim, errCh chan error) error {
 
 	g.ChannelExf("let s:gopls_logfile=%q", logfile.Name())
 
-	gopls := exec.Command(g.goplspath, "-rpc.trace", "-logfile", logfile.Name())
+	goplsArgs := []string{"-rpc.trace", "-logfile", logfile.Name()}
+	if flags := strings.Fields(os.Getenv(string(config.EnvVarGoplsFlags))); flags != nil {
```

Comment:
I think we should look to handle quoted strings for values that can include spaces. Not because I have an immediate use case for it, but because a) people will expect it and b) there's an implementation we can use.

Take a look at https://github.com/golang/go/blob/af880809d8c3896a7d37d7547c290cd9bc34530f/src/cmd/go/internal/generate/generate.go#L332

Reply:
Added